src/Nordea/Components/Status.elm

```diff
@@ -0,0 +1,77 @@
+module Nordea.Components.Status exposing (blue, gray, green, red, yellow)
+
+import Css exposing (Color, backgroundColor, borderRadius, display, ellipsis, fontSize, hidden, inlineBlock, lineHeight, maxWidth, overflow, padding, pct, rem, textOverflow)
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes exposing (css)
+import Nordea.Resources.Colors as Color
+
+
+type StatusColor
+    = Green
+    | Red
+    | CloudBlue
+    | Yellow
+    | LightGray
+
+
+view : String -> StatusColor -> Html msg
+view text statusColor =
```

Comment:
I'm a fan of this type of interface. It allows setting arbitrary attributes (e.g., adding a custom css margin, etc.), and allows the content to be set to custom html. Setting custom html instead of a String is handy in cases where we want to set some custom styling to the text, for example. 

```suggestion
view : List (Attribute msg) -> List (Html msg) -> StatusColor -> Html msg
view attrs children statusColor =
```

Reply:
According to Espen and Thea it should not be possible to change the styling and content ☺️